ichnaea/db.py

```diff
@@ -17,36 +16,47 @@
 from ichnaea.conf import settings
 
 
-DB_TYPE = {
-    "ddl": settings("db_ddl_uri"),
-    "ro": settings("db_readonly_uri"),
-    "rw": settings("db_readwrite_uri"),
-}
+DB_TYPE = {"ro": settings("db_readonly_uri"), "rw": settings("db_readwrite_uri")}
 
 
-def get_alembic_config():
-    cfg = AlembicConfig()
+class SqlAlchemyUrlNotSpecified(Exception):
+    """Raised when SQLALCHEMY_URL is not specified in environment."""
 
-    script_location = os.path.join(os.path.dirname(__file__), "alembic")
-    assert os.path.exists(script_location)
-    cfg.set_section_option("alembic", "script_location", script_location)
-    cfg.set_section_option("alembic", "sqlalchemy.url", settings("db_ddl_uri"))
-    return cfg
+    def __init__(self, *args, **kwargs):
+        super().__init__("SQLALCHEMY_URL is not specified in the environment")
```

Comment:
The first argument is `args[0]` which gets used as the msg. I figure this lets me (ab)use the exception type, but have a helpful message regardless of where it's raised.